.github/workflows/pr.yaml

```diff
@@ -0,0 +1,23 @@
+name: Pull Request
+on: pull_request
+permissions:
+  contents: read
+jobs:
+  lint:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - uses: golangci/golangci-lint-action@v2
+      with:
+        version: v1.41.1
+      continue-on-error: true # temporary
+  buildtest:
```

Comment:
good to seperate build from test command.

Reply:
Done.